files/build_templates/sonic_debian_extension.j2

```diff
@@ -132,8 +132,7 @@ sudo bash -c "echo dhcp_as_static=true >> $FILESYSTEM_ROOT/etc/sonic/updategraph
 sudo bash -c "echo enabled=false > $FILESYSTEM_ROOT/etc/sonic/updategraph.conf"
 {% endif %}
 {% if shutdown_bgp_on_start == "y" %}
-sudo bash -c "echo bgp_admin_state: > $FILESYSTEM_ROOT/etc/sonic/bgp_admin.yml"
-sudo bash -c "echo '  all: off' >> $FILESYSTEM_ROOT/etc/sonic/bgp_admin.yml"
+sudo bash -c "echo '{ \"bgp_admin_state\": { \"all\": false }  }' >> $FILESYSTEM_ROOT/etc/sonic/config_db.json"
 {% endif %}
```

Comment:
I don't think it's a good idea to tell whether configuration is loaded or not by looking at the content of config. Instead, we should find other ways to ensure initial config loading is finished before other container consume DB data, either by systemd service sequence or by setting flag in DB.

---
In reply to: [128321125](https://github.com/Azure/sonic-buildimage/pull/808#discussion_r128321125) [](ancestors = 128321125)